metis-dereference/metis-dereference-import/src/main/java/eu/europeana/metis/dereference/vocimport/utils/DereferenceValidationUtils.java

```diff
@@ -0,0 +1,64 @@
+package eu.europeana.metis.dereference.vocimport.utils;
+
+import edu.emory.mathcs.backport.java.util.Arrays;
+import eu.europeana.metis.dereference.vocimport.exception.VocabularyImportException;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Utils class focused on validating values used in dereferencing
+ */
+public class DereferenceValidationUtils {
+
+    private static final String PROPERTY_FIELD_NAME = "valid.directories.values";
+    private String dereferencingPropertiesLocation =
+            "metis-dereference/metis-dereference-rest/src/main/resources/dereferencing.properties";
```

Comment:
We shouldn't be doing that here. We are hardcoding the location of the file.
Instead, the property should be read where it's used, in this case the dereferencing rest(like the rest of the values). 
And the white listed url prefixes should be propagated here.

Reply:
I was struggling in the beginning on how to do that, but after doing some search, I believe I found a way to do exactly this. I will changes this

Reply:
I have changed this part of the code. I've used spring annotations to handle this